run/data/data.go

```diff
@@ -76,6 +78,12 @@ type Column struct {
 	Orig         interface{} `yaml:"-" json:"-"` // the raw database column data
 }
 
+// Index is the data about a table index.
+type Index struct {
+	DBName  string  // dbname of the index
```

Comment:
I would give the Index both a DBName and a converted Name. I don't know that anyone will use either of them, BUT, it doesn't hurt anything, and it keeps the code consistent.